lib/keen/aes_helper_old.rb

```diff
@@ -40,11 +41,15 @@ def unhexlify(msg)
       end
 
       def pad(msg)
-        pad_len = BLOCK_SIZE - (msg.length % BLOCK_SIZE)
+        missing_chars = msg.length % BLOCK_SIZE
+        return msg if missing_chars == 0
```

Comment:
This is the ruby style I've seen/used in the past, and is also a style we're adopting in JS